### ISSUE TITLE
is it a bug or supposed behaviour?

### ISSUE THREAD

**Issue by plandem** [Hypercontext/linkifyjs#385]
https://github.com/Hypercontext/linkifyjs/blob/3eb5b8aacc286e57e220cd6ce2d9c139d96e7322/packages/linkifyjs/src/core/fsm.js#L79-L81

```
nextState.jr.append(templateState.jr);
nextState.jr = templateState.jd; // <- should not it be `nextState.jd` here?
nextState.t = token || templateState.t;
```

**Comment by nfrasser** [comment]
Hi @plandem, great catch, this is a bug. It's fixed in the latest `v4.0.0-beta.5` (install with `npm install linkifyjs@beta`)

**Comment by plandem** [comment]
Is it possible to fix v3?
